pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "pyds8k"
+description = "DS8000 Python Client"
+dynamic = ["version"]
+readme = "README.md"
+license-files = ["LICENSE"]
+requires-python = ">=3.9"
+authors = [
+    { name = "Zhang Wu", email = "shwzhang@cn.ibm.com"},
```

Comment:
should we remove Zhang as an author at this point?   Not even sure they're still at IBM....couldn't find that email address.